wire-library/wire-moshi-adapter/src/main/java/com/squareup/wire/WireJsonAdapterFactory.kt

```diff
@@ -135,10 +135,10 @@ class WireJsonAdapterFactory private constructor(
       @Throws(IOException::class)
       override fun toJson(writer: JsonWriter, value: Long?) {
         if (value!! < 0) {
-          val unsigned = power64.add(BigInteger.valueOf(value))
+          val unsigned = power64.add(BigInteger.valueOf(value)).toString()
```

Comment:
I don’t think this is correct

Reply:
Though perhaps in this incorrectness it is consistent with proto3? What do the docs advise?